packages/neuron-ui/src/components/NetworkSetting/index.tsx

```diff
@@ -86,9 +86,15 @@ const NetworkSetting = ({ chain = chainState, settings: { networks = [] } }: Sta
     }
   }, [currentId, networks])
 
+  const isInternalLightClient = (id: string) => id.includes('light_client')
```

Comment:
it is recommended to paste a URL here to link to the source of the constant `light_client`